frontend/src/components/navigation/MenuItems.js

```diff
@@ -0,0 +1,34 @@
+export const MenuItems = [
+  {
+      title: 'Alumni',
+      url: '#',
+      cName: 'nav-links'
```

Comment:
Remove the `cName` as we're using styled components, and actually fill in the `url` with the relevant value (`/alumni`, for example).